### PR TITLE
Added FAQ about app not Uninstalling

### PR DESCRIPTION
**Pull request by joseph619**

### CODE REVIEW

File: src/help/faq/README.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ You can change language by going to <Navigation item="more"/> → <Navigation it
 :::
 
 ### Miscellaneous
+
+#### I can't uninstall Tachiyomi
+On **Android 10**, a bug was introduced which prevented users from uninstalling the app if the phone had a MicroSD card. To fix the issue, please remove the MicroSD card from phone first, then uninstall Tachiyomi.
```
Comment (GHOSTBEAR) [inline src/help/faq/README.md:91]:
```suggestion
On **Android 10**, a bug was introduced which prevented users from uninstalling the app if the phone had a MicroSD card. To fix the issue, please remove the MicroSD card from the phone first, then uninstall Tachiyomi.
```
You are just missing a `the` before `phone` in `To fix the issue, please remove the MicroSD card from phone first`

**Comment by specterflare** [comment]
Could you turn the header into a question, like "Why can't I uninstall Tachiyomi?"? I'm working on #236 and I think it would be best if the site was consistent.

File: src/help/faq/README.md (hunk 1 above)
Comment (joseph619) [reply src/help/faq/README.md]: Done

**Comment by joseph619** [comment]
> Could you turn the header into a question, like "Why can't I uninstall Tachiyomi?"? I'm working on #236 and I think it would be best if the site was consistent.

Done

**Pull request merged**